recipe/meta.yaml

```
@@ -28,13 +28,14 @@ requirements:
     - setuptools
     - wheel
     - numpy {{ numpy }}
+    - packaging >20.9
   run:
     - python
-    - numpy
+    - {{ pin_compatible('numpy') }}
     - scipy
     - scikit-learn
     - pandas
-    - tqdm >4.25.0
+    - tqdm >4.27.0
```

```suggestion
    - tqdm >=4.27.0
```
https://github.com/shap/shap/blob/v0.42.1/pyproject.toml#L19

---

Yeap, missed this, thanks!